core/types/vote.go

```diff
@@ -12,35 +12,41 @@ import (
 const (
 	BLSPublicKeyLength = 48
 	BLSSignatureLength = 96
+
+	MaxAttestationExtraLength = 256
 )
 
 type BLSPublicKey [BLSPublicKeyLength]byte
 type BLSSignature [BLSSignatureLength]byte
 type ValidatorsBitSet uint64
 
+// VoteData represents the vote data that validators vote for fast finality.
```

Comment:
"voteData represents vote data"... emm, sounds the words duplicated. Maybe we can make it more sense, like VoteData represents brief blockInfo inside the vote, SourceNumber, hash, etc.

Reply:
Sorry, I don't think in the comment we should use SouceNumber these variables, they are not readable, and I comment on each field in the VoteData.